Automattic/wp-calypso
# Themes showcase: Add Full Site Editing beta tab

#### Changes proposed in this Pull Request

* Add Full Site Editing beta tab to themes showcase

![image](https://user-images.githubusercontent.com/2256104/126312107-90c6b74b-233a-458f-a9bc-26f835d299fc.png)


#### Testing instructions



* Navigate to Appearance -> Themes
* Make sure Full Site Editing Beta tab is available
* Confirm full site editing tab only shows themes with full site editing feature



Related to #https://github.com/Automattic/wp-calypso/issues/54508
